Feat/nawin/userdata

nawinpoli opened the pull request:
Add userdata and userdata component.

bunnybunbun37204 committed:
ci: setup output to ssg

nawinpoli committed:
feat(userdata): add userdata

siwadolrungmart commented on src/components/elements/Segmentcontrol.tsx:
```diff
@@ -0,0 +1,54 @@
+        <Input name = "id_expired_date" type="text" className='w-full px-4 py-2 border border-[#c34e5e] rounded-md placeholder:text-[#e4b0b9]  focus:border-pink-400 mt-2' placeholder='วันหมดอายุบัตรประชาชน'/>
+        <Input name = "back_id" type="text" className='w-full px-4 py-2 border border-[#c34e5e] rounded-md placeholder:text-[#e4b0b9]  focus:border-pink-400 mt-2' placeholder='หมายเลขหลังบัตรประชาชน'/>
+        <Button type="submit" className='w-full py-2 text-white bg-[#c34e5e] rounded-md hover:bg-pink-700 focus:outline-none mt-4' >ต่อไป</Button>
+      </form>
```
1.อยากให้ input เก็บค่าได้ด้วย 
2.จัดหน้าให้หน่อยนะมันยังไม่ตรง figma 
3.หมายเลขประชาชนยังไม่เป็น int นะ กำหนดให้หน่อยอย่าให้เกิน13หลัก
4.เปลี่ยน tag สีตาม ci หน่อย
5.คำนำหน้ากดแล้วยังไม่เปลี่ยนสีนะ

siwadolrungmart commented on public/Vector1.svg:
เอาไฟล์รูปไปไว้ใน assets ให้หน่อย

siwadolrungmart commented on src/components/elements/Segmentcontrol.tsx:
```diff
@@ -0,0 +1,54 @@
+} from "@/components/ui/menubar"
+
+const SegmentedControl = () => {
+  const [selected, setSelected] = useState('');
```
ถ้าไม่ใช้เอาออกให้หน่อย

nawinpoli committed:
fix(datepicker add,personal id is int and lenght<13,prefixch): datapicker: personalid:input_collect

data picker add, personalid is is int lenght<13 , input data collect ,Prefix นาย นาง นางสาว
(เหลือปรับรูปหน้ากับเปลี่ยนฟอนต์)

nawinpoli committed:
fix(input handlesubmit): prefix,picture asset

prefix choose , move picture to assets,personal id to int and lenght<13,handlesubmit add

nawinpoli committed:
test(save change before pull): save change before pull

save change before pull

nawinpoli committed:
fix(some fix): จัดรูปหน้า

จัดดรูปหน้า เหลือเปลี่ยนฟอนต์เค้าเปลี่ยนไม่เป็น
